lib/parse.coffee

```diff
@@ -76,6 +76,10 @@ exports.parseOptions = (definedOptions, options = {}) ->
 				throw new Error("Option #{definedOption.signature} is required")
 
 		continue if not definedOption.matches(value)
-		result[signature] = _.parseInt(value) or value
+
+		if /^\d+$/.test(value)
```

Comment:
Capitano parses it as strings, which I think is good enough for now. We can add support if needed later on.